samplers/parser.go

```diff
@@ -221,7 +221,7 @@ func ParseEvent(packet []byte) (*UDPEvent, error) {
 	if len(textChunk) != textExpectedLength {
 		return nil, errors.New("Invalid event packet, actual text length did not match encoded length")
 	}
-	ret.Text = string(textChunk)
+	ret.Text = strings.Replace(string(textChunk), "\\n", "\n", -1)
```

Comment:
Right, I'm asking if the actual string we're searching is expected to have multiple instances of the escaped newline.